Schenng/colour-wave-android
# New asset for splash screen (fix)

Splash screen image was off centre.  
uploaded a new image. 

To test: 
1. Try cold start on master. Notice space around circles and edge of the screen is not symmetrical
2. Try cold start on this branch to see symmetrical splash screen
